CHANGELOG/CHANGELOG-1.4.0.md

```diff
@@ -1,5 +1,5 @@
 - [v1.4.0](#v140)
-  - [Changelog since v1.3.1](#changelog-since-v130)
+  - [Changelog since v1.3.0](#changelog-since-v130)
```

Comment:
Should be [changelog since v1.3.1](https://github.com/dell/csm/blob/c68ca76fc1c3697b00fd4e141764a038f5cfeb72/CHANGELOG/CHANGELOG-1.3.0.md#v131)